components/molecule/modal/src/HoC/WithUrlState.js

```diff
@@ -0,0 +1,76 @@
+import React, {useEffect, useState} from 'react'
+import PropTypes from 'prop-types'
+
+const getUrlWithHash = hashName => {
```

Comment:
What about `createUrlWithHash` or `generateUrlWithHash`?

Reply:
When you call this method you should be aware of how / where this method get the url with hash. Using the names you propose, you are coupling the implementation to the name.

Reply:
Since we can generate the URL without hash, what about calling it just `getUrl`?